recipes/faker/meta.yaml

```diff
@@ -0,0 +1,267 @@
+    - faker.providers.person.ja_JP
+    - faker.providers.person.ko_KR
+    - faker.providers.person.lt_LT
+    - faker.providers.person.lv_LV
+    - faker.providers.person.ne_NP
+    - faker.providers.person.nl_NL
+    - faker.providers.person.no_NO
+    - faker.providers.person.pl_PL
+    - faker.providers.person.pt_BR
+    - faker.providers.person.pt_PT
+    - faker.providers.person.ru_RU
+    - faker.providers.person.sl_SI
+    - faker.providers.person.sv_SE
+    - faker.providers.person.tr_TR
+    - faker.providers.person.uk_UA
+    - faker.providers.person.zh_CN
+    - faker.providers.person.zh_TW
+    - faker.providers.phone_number
+    - faker.providers.phone_number.bg_BG
+    - faker.providers.phone_number.bs_BA
+    - faker.providers.phone_number.cs_CZ
+    - faker.providers.phone_number.de_DE
+    - faker.providers.phone_number.dk_DK
+    - faker.providers.phone_number.el_GR
+    - faker.providers.phone_number.en_AU
+    - faker.providers.phone_number.en_CA
+    - faker.providers.phone_number.en_GB
+    - faker.providers.phone_number.en_US
+    - faker.providers.phone_number.es_ES
+    - faker.providers.phone_number.es_MX
+    - faker.providers.phone_number.fa_IR
+    - faker.providers.phone_number.fi_FI
+    - faker.providers.phone_number.fr_CH
+    - faker.providers.phone_number.fr_FR
+    - faker.providers.phone_number.hi_IN
+    - faker.providers.phone_number.hr_HR
+    - faker.providers.phone_number.it_IT
+    - faker.providers.phone_number.ja_JP
+    - faker.providers.phone_number.ko_KR
+    - faker.providers.phone_number.lt_LT
+    - faker.providers.phone_number.lv_LV
+    - faker.providers.phone_number.ne_NP
+    - faker.providers.phone_number.nl_BE
+    - faker.providers.phone_number.nl_NL
+    - faker.providers.phone_number.no_NO
+    - faker.providers.phone_number.pl_PL
+    - faker.providers.phone_number.pt_BR
+    - faker.providers.phone_number.pt_PT
+    - faker.providers.phone_number.ru_RU
+    - faker.providers.phone_number.sk_SK
+    - faker.providers.phone_number.sl_SI
+    - faker.providers.phone_number.sv_SE
+    - faker.providers.phone_number.tr_TR
+    - faker.providers.phone_number.uk_UA
+    - faker.providers.phone_number.zh_CN
+    - faker.providers.phone_number.zh_TW
+    - faker.providers.profile
+    - faker.providers.profile.en_US
+    - faker.providers.python
+    - faker.providers.python.en_US
+    - faker.providers.ssn
+    - faker.providers.ssn.en_CA
+    - faker.providers.ssn.en_US
+    - faker.providers.ssn.fi_FI
+    - faker.providers.ssn.fr_CH
+    - faker.providers.ssn.hr_HR
+    - faker.providers.ssn.it_IT
+    - faker.providers.ssn.ko_KR
+    - faker.providers.ssn.nl_BE
+    - faker.providers.ssn.nl_NL
+    - faker.providers.ssn.pt_BR
+    - faker.providers.ssn.ru_RU
+    - faker.providers.ssn.sv_SE
+    - faker.providers.ssn.uk_UA
+    - faker.providers.ssn.zh_CN
+    - faker.providers.ssn.zh_TW
+    - faker.providers.user_agent
+    - faker.providers.user_agent.en_US
+    - faker.utils
```

Comment:
This import list looks really long. Maybe we should move it to a `run_test.py` file to keep `meta.yaml` lean and mean.